src/metabase/driver/bigquery.clj

```diff
@@ -473,8 +473,8 @@
           ;; people can manually specifiy "foreign key" relationships in admin and everything should work correctly.
           ;; Since we can't infer any "FK" relationships during sync our normal FK tests are not appropriate for BigQuery, so they're disabled for the time being.
           ;; TODO - either write BigQuery-speciifc tests for FK functionality or add additional code to manually set up these FK relationships for FK tables
-          :features              (constantly (when-not config/is-test?
-                                               #{:foreign-keys}))
+          :features              (constantly (cond-> #{:native-parameters}
+                                                     (not config/is-test?) :foreign-keys))
```

Comment:
what are you trying to accomplish here? This code doesn't make sense and will break foreign key support for BigQuery. e.g. 

```clojure
(cond-> #{:native-parameters}
  true :foreign-keys) 
;; -> nil

(cond-> #{:native-parameters}
  false :foreign-keys)
;; -> #{:native-parameters}
```

```

:-1: